src/sentry/integrations/atlassian_connect.py

```diff
@@ -56,14 +56,21 @@ def get_integration_from_jwt(token, path, query_params, method='GET'):
     # by the add-on during the installation handshake
     try:
         integration = Integration.objects.get(
-            provider='jira',
+            provider=provider,
             external_id=issuer,
         )
     except Integration.DoesNotExist:
         raise AtlassianConnectValidationError('No integration found')
     # Verify the signature with the sharedSecret and
     # the algorithm specified in the header's alg field.
-    decoded_verified = jwt.decode(token, integration.metadata['shared_secret'])
+
+    # If it's BitBucket, don't verify audience
```

Comment:
I feel like it would be helpful to say a bit about why. I'm not sure there is a why in this case... maybe not.

Reply:
Agreed, I'd like to know why

Reply:
@EvanPurkhiser while I pointed it out. I'm wondering if it's literally just because the guy at BB told us to. Uhm I'm actually not sure if it's just what BB created to satisfy a request we made and so it's a special case or if there's an underly reason as to why. I don't understand jwt enough to answer it though